src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -1184,6 +1184,7 @@ def test_postgres_flexible_server_vnet_mgmt_supplied_vname_and_subnetname(self,
     def test_mysql_flexible_server_vnet_mgmt_supplied_vname_and_subnetname(self, resource_group):
         self._test_flexible_server_vnet_mgmt_supplied_vname_and_subnetname('mysql', resource_group)
 
+    @live_only()
```

Comment:
May I know why this test need to be live only? Shall I involve RDBMS team contributor to fix the test?

Reply:
I already contacted them via Teams and @alanenriqueo will work on that.